crates/core/src/bc/xml.rs

```diff
@@ -1566,6 +1569,48 @@ pub struct Schedule {
     pub time_block_list: TimeBlockList,
 }
 
+/// List of users
+#[derive(PartialEq, Eq, Default, Debug, Deserialize, Serialize)]
+pub struct UserList {
+    /// XML Version
+    #[serde(rename = "@version")]
+    pub version: String,
+    /// The actual user-list
+    #[serde(rename = "User", skip_serializing_if = "Option::is_none")]
+    pub user_list: Option<Vec<User>>,
+}
+
+/// A struct for reading and writing camera user records
+#[derive(PartialEq, Eq, Default, Debug, Deserialize, Serialize)]
+pub struct User {
+    /// The user_name is used to identify the user in the API
+    #[serde(rename = "userName")]
+    pub user_name: String,
+    /// The password seems to only be included when creating or modifying a user
+    #[serde(rename = "password", skip_serializing_if = "Option::is_none")]
+    pub password: Option<String>,
```

Comment:
```suggestion
    #[serde(rename = "password", skip_serializing_if = "Option::is_none", skip_deserializing)]
    pub password: Option<String>,
```

Let's add a skip_deserializing field. That way if my E1 sends a password we ignore it and don't keep it in memory or send it back to the camera. I want to see if it will work without that pass field on my camera, since it's most likely ignored unless the `userSetState` is `modify` anyways

Reply:
Update: Confirmed it works without the password field on my old E1